plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/menu/ViewMenu.java

```diff
@@ -64,7 +68,18 @@ public void select() {
 		if (item == null) {
 			mh.select(menuItem);
 		} else {
-			mh.select(item);
+			if ((item.getAction().getStyle() == IAction.AS_CHECK_BOX) || (item.getAction().getStyle() == IAction.AS_RADIO_BUTTON)) {
```

Comment:
Looks good but we prefer all sync and async operations to be executed within Handler classes. Could you please move this whole logic to MenuHandler class https://github.com/tsedmik/reddeer/blob/REDDEER-1359/plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/MenuHandler.java#L70 ? Thanks!